lang/en.json

```diff
@@ -4747,6 +4706,54 @@
     "Legacy": "Legacy Rules (2014)",
     "Modern": "Modern Rules (2024)"
   },
+  "VARIANT": {
+    "Hint": "Enable and configure rules variations offered by the system.",
+    "Label": "Configure Variant Rules",
+    "Name": "Variant Rules",
+    "AllowFeats": {
+      "Hint": "Allow players to choose a feat rather than taking an ability score improvement during class advancement when using the Legacy rules.",
+      "Name": "Allow Feats"
+    },
+    "CurrencyWeight": {
+      "Hint": "Carried currency affects character encumbrance.",
+      "Name": "Track Currency Weight"
+    },
+    "Encumbrance": {
+      "Hint": "Enable automatic tracking of encumbrance and the application of status effects for characters carrying too much.",
+      "Name": "Encumbrance Tracking",
+      "None": "None",
+      "Normal": "Normal (max carrying capacity)",
+      "Variant": "Variant (encumbered & heavily encumbered)"
+    },
+    "HonorScore": {
+      "Hint": "Enable the use of the optional Honor ability score. Requires the world to be reloaded.",
```

Comment:
Do we need these 'required world to be reloaded' hints anymore? I can't remember if we added them before we had `requiresReload`, or whether we added them in addition to that dialog.

Reply:
These hints were here from the start when honor & sanity were added when we were using our own debounced reload.